.github/workflows/push.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         build_cmd:
         - ./build-latest.sh
         - PRERELEASE=true ./build-latest.sh
-        - ./build-next.sh
```

Comment:
What is the reason you did not remove the script as well? Do we still need it?

Reply:
At the moment the script is not needed. I guess, if we need it once more, we can dig it out of the commit history. 
I will remove it.